### DISCUSSION TITLE
stream music with websocket

### DISCUSSION THREAD

**Discussion by hypy13** [fastapi/fastapi#7332] [Questions] [answered]
is it possible to stream music with tcp websocket?

i want to chunk a music to 10 pieces and each time send them to client
and if client  seek the music the server send from the second requested
i need this for low latency

thanks for your help

**Comment by includeamin** [comment]
what is your use-case?
is that live streaming or offline streaming?

**Comment by hypy13** [comment, marked as answer]
@includeamin
its online streaming 
i want to stream music to client by streaming like spotify
after searching a lot i found hls (HTTP LIVE STREAMING) and its good for now

by the way hls is ok im going to close the issue

**Comment by includeamin** [comment]
> @includeamin
> i want to stream music to client by streaming like spotify
> after searching a lot i found hls (HTTP LIVE STREAMING) and its good for now

I wanted to say exactly the same things :)).  HLS, Dash .etc

**Comment by tiangolo** [comment]
Thanks for the help @includeamin ! ☕ 

And thanks @hypy13 for coming back to close the issue.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
